import-scripts/pipelines/automation-environment.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
Note to self : This location is not the deployment location ... the deployer should copy/move the appropriate script into the upper level scripts directory (selecting the correct one based on which platform we are deploying to)